GameEngineTests/PlayerHandTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using GameEngine;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace GameEngineTests
+{
+    [TestClass]
+    public class PlayerHandTests
+    {
+        [TestMethod]
+        public void ShouldDiscard()
+        {
+            var hand = new PlayerHand();
+            hand.Add(CardType.Blue);
```

Comment:
💡 What do you think about constructing the `PlayerHand` with the blue card in it, so we're only testing `Discard` here (and not also `Add`)?